terraform/datadog/concourse.tf

```diff
@@ -52,6 +52,24 @@ resource "datadog_monitor" "continuous-smoketests-failures" {
   tags = ["deployment:${var.env}", "service:${var.env}_monitors"]
 }
 
+resource "datadog_monitor" "concourse-btrfs" {
+  name                = "${format("%s concourse excess btrfs processes running", var.env)}"
+  type                = "query alert"
+  message             = "Large number of btrfs processes running. Concourse probably needs shaking."
+  escalation_message  = "Large number of btrfs processes still running. Concourse probably needs shaking."
```

Comment:
Let's remove it from this monitor so as not to further the pattern.

Up to you about whether we remove it from all the others. It should probably be a separate PR if we do.